test/test_ops.py

```diff
@@ -1000,5 +1006,34 @@ def gen_iou_check(box, expected, tolerance=1e-4):
             gen_iou_check(box_tensor, expected, tolerance=0.002 if dtype == torch.float16 else 1e-3)
 
 
+class TestStochasticDepth:
+    @pytest.mark.skipif(stats is None, reason="scipy.stats not available")
```

Comment:
As a slightly cleaner pattern we could use: `pexpect = pytest.importorskip("pexpect")` in the test body https://docs.pytest.org/en/latest/how-to/skipping.html#summary

But is there a reason we should not expect stats to be available here? I think we rely on scipy unconditionally in other tests

Reply:
I'm using the same pattern we use here:
https://github.com/pytorch/vision/blob/3fa2055b5a4807a8ffc577c53da180382cafd17c/test/test_transforms.py#L17-L20

https://github.com/pytorch/vision/blob/3fa2055b5a4807a8ffc577c53da180382cafd17c/test/test_transforms.py#L512

Perhaps we could create a ticket to refactor all of them?

Reply:
sure, but we might as well start using his already :)
We started using it in https://github.com/pytorch/vision/blob/master/test/test_onnx.py#L20 to skip an entire test file